app/upgrades.go

```diff
@@ -216,6 +217,20 @@ func (app *StrideApp) setupUpgradeHandlers(appOpts servertypes.AppOptions) {
 		),
 	)
 
+	// v17 upgrade handler
+	app.UpgradeKeeper.SetUpgradeHandler(
```

Comment:
Is PFM the only module we're adding for v17? Need to set the store key(s)

Reply:
yes and added